qdev_wrappers/T3/customised_instruments.py

```diff
@@ -322,8 +241,7 @@ def _set_fine_voltage(self, voltage):
             fine_chan = 3
         else:
             raise RuntimeError("Fine mode only works for Chan 0 and 1")
-        coarse_part = self._dac_code_to_v(
-            self._dac_v_to_code(round(voltage,2)-0.01) )
+        coarse_part = self._dac_code_to_v(self._dac_v_to_code(voltage-0.001))
```

Comment:
@ThorvaldLarsen I think this should not be changed... I don't know how this got in here. But as I remember it, does the originalversion with the rounding come from T3. Can you confirm this?

Reply:
I think this change is good. It avoids large voltage jumps in between setting the coarse and fine parts of the voltage. Not sure how it got here but reading it I like the change.